CMakeLists.txt

```diff
@@ -282,7 +282,9 @@ install(FILES ${ANTARES_SOLVER_PATH}
 if(CMAKE_SYSTEM_NAME MATCHES "Linux")
     get_linux_lsb_release_information()
     message(STATUS "Linux ${LSB_RELEASE_ID_SHORT} ${LSB_RELEASE_VERSION_SHORT} ${LSB_RELEASE_CODENAME_SHORT}")
-    set(CPACK_SYSTEM_NAME "${LSB_RELEASE_ID_SHORT}-${LSB_RELEASE_VERSION_SHORT}") 
+    set(CPACK_SYSTEM_NAME "${LSB_RELEASE_ID_SHORT}-${LSB_RELEASE_VERSION_SHORT}")
+else()
+    set(CPACK_SYSTEM_NAME "windows")
```

Comment:
According to CMake's documentation, CPACK_SYSTEM_NAME is `win32` or `win64` by default. Is there any reason to set it to "windows" instead ?

Reply:
Good point !
CPack was included **after** `CPACK_SYSTEM_NAME ` variable was not defined. 
CPack is now include **before** `CPACK_SYSTEM_NAME  `use